clustered/server/src/main/java/org/ehcache/clustered/server/management/PoolSettingsManagementProvider.java

```diff
@@ -43,7 +43,7 @@
   public Collection<Descriptor> getDescriptors() {
     Collection<Descriptor> descriptors = super.getDescriptors();
     descriptors.add(new Settings()
-      .set("type", "PoolSettingsManagementProvider")
+      .set("type", getCapabilityName())
```

Comment:
@mathieucarbou right, every change on the TMS backend topology wil have an impact on the TMS / TMC
I'd like to have the TMS ready for those changes before I approve; but I won't be super available this week; I'll try my best.